docs/howtoguides/enable_landscape.rst

```diff
@@ -0,0 +1,47 @@
+How to enable Landscape
+***********************
+
+``pro`` offers a convenient way to register a machine with Landscape via the
+``pro enable landscape`` command.
```

Comment:
`pro` offers convenient ways of accessing all the Pro services ;) can we remove that bit and just leave "you can register a machine with Landscape via the ``pro enable landscape`` command. Then, let's add a bit about the two different options (interactive/non-interactive) for enabling so that we can add a couple of the keywords that might lead people to find the page if they're searching for it (like hands off/automatic, that sort of thing).

Reply:
Probably also an idea to state up front what info they'll need to complete the registration (which I assume is the same info no matter which method they choose?)

Reply:
Another question that users might have (which occurred to me after I saw the 40k machines thing added to the index) - how do I register more than a few machines at once? Doing this one by one for an entire fleet would be quite inefficient so is there some method we recommend for doing this at larger scale?

Reply:
> `pro` offers convenient ways of accessing all the Pro services ;) can we remove that bit and just leave "you can register a machine with Landscape via the `pro enable landscape` command. Then, let's add a bit about the two different options (interactive/non-interactive) for enabling so that we can add a couple of the keywords that might lead people to find the page if they're searching for it (like hands off/automatic, that sort of thing).

Sounds good, I'll remove/add these.

> Probably also an idea to state up front what info they'll need to complete the registration (which I assume is the same info no matter which method they choose?)

Good idea, I'll add it.

> how do I register more than a few machines at once?

This is out of scope for pro-client. The correct answer is dependent on the customer's infrastructure setup. We can add a line about the non-interactive case being useful for automation, but it is up to the customer to run the command on many machines.